corehq/apps/fixtures/templates/fixtures/manage_tables.html

```diff
@@ -38,6 +38,20 @@
            data-bind="click: downloadExcels, attr: {'disabled': selectedTables().length < 1}">
           <i class="fa fa-cloud-download"></i> {% trans "Download Lookup Tables" %}
         </a>
+        <!-- ko if: hasLinkedModels -->
+        <span data-bind="ifnot: unlockLinkedData">
```

Comment:
convention is to use hidden elements, not spans, for pure conditionals. Also, it is not valid HTML to wrap `button` elements in `spans`. in fact, `spans` are really low on the tier that they shouldn't be wrapping anything.

```
<!-- ko ifnot: unlockLinkedData --><!-- /ko -->
```

Reply:
Updated. Out of curiosity, do you have a link that lays out buttons within spans as not valid HTML? The [spec](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element) seems to suggest that, as both are considered 'phrasing content', that it's just preference.